code/__DEFINES/is_helpers.dm

```diff
@@ -1,3 +1,5 @@
+#define isdatum(thing) (istype(thing, /datum))
+
```

Comment:
This isn't used. Also this would return true for all atoms because everything is a datum :P
```suggestion
```

Reply:
Indeed, seems like junk I forgot to remove. I did use it in some previous iteration before xd